src/tax-lot/tax-lot.controller.ts

```diff
@@ -36,44 +36,42 @@ export class TaxLotController {
   constructor(private readonly taxLotService: TaxLotService) {}
 
   @Get("/:bbl")
-  @UsePipes(new ZodValidationPipe(getTaxLotByBblPathParamsSchema))
-  async findDetailsByBbl(@Param() params: GetTaxLotByBblPathParams) {
-    return this.taxLotService.findTaxLotByBbl(params.bbl);
+  @UsePipes(new ZodValidationPipe(findTaxLotByBblPathParamsSchema))
+  async findDetailsByBbl(@Param() params: FindTaxLotByBblPathParams) {
+    return this.taxLotService.findByBbl(params.bbl);
   }
 
   @Get("/:bbl/geojson")
-  @UsePipes(new ZodValidationPipe(getTaxLotGeoJsonByBblPathParamsSchema))
+  @UsePipes(new ZodValidationPipe(findTaxLotGeoJsonByBblPathParamsSchema))
   async findTaxLotByBblGeoJson(
-    @Param() params: GetTaxLotGeoJsonByBblPathParams,
+    @Param() params: FindTaxLotGeoJsonByBblPathParams,
   ) {
-    return this.taxLotService.findTaxLotByBblGeoJson(params.bbl);
+    return this.taxLotService.findByBblGeoJson(params.bbl);
```

Comment:
From Guideline 6 of discussion #115 

>     6. Non-default formats should be affixed to the "object" in the variable name
>        
>        * If getting the geojson of zoning districts by a tax lot bbl, it should be `findZoningDistrictsGeoJsonByBbl`
>        * If getting the geojson format of a tax lot within the tax lot domain, the domain is excluded like so: `findGeoJsonByBbl`
>        * If getting a list of geojson within the domains home, the object is "Many", like so: `findManyGeoJson`
>        * The same rule would apply if we started supporting formats like `csv`
```suggestion
    return this.taxLotService.findGeoJsonByBbl(params.bbl);
```